docs/rules/no-array-reduce.md

```diff
@@ -9,7 +9,7 @@
 
 It's only somewhat useful in the rare case of summing up numbers, which is allowed by default.
 
-Use `eslint-disable` comment if you really need to use it.
+Use `eslint-disable` comment if you really need to use it or disable the rule entirely if you prefer functional programming.
```

Comment:
You can't change the mind of functional programmers, so it's best to make it clear this is not for them.